src/OpenLoco/Ui/ProgressBar.cpp

```diff
@@ -0,0 +1,123 @@
+#include "ProgressBar.h"
+#include "../Interop/Interop.hpp"
+#include "../Localisation/FormatArguments.hpp"
+#include "../Localisation/StringManager.h"
+#include "../OpenLoco.h"
+#include "WindowManager.h"
+
+using namespace OpenLoco::Interop;
+
+namespace OpenLoco::Ui::ProgressBar
+{
+    static constexpr uint32_t _isInternalWindow = (1 << 31);
+    static uint32_t _1136590;
```

Comment:
We can just name this _isInternalWindow and make it a bool as we now use all of its callers.